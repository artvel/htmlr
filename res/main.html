<!DOCTYPE html>
<html lang="en">
<head></head>
<body>
    {% include "sub/sub-1.html" %}
</body>
</html>