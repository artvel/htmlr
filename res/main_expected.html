<!DOCTYPE html>
<html lang="en">
<head></head>
<body>
    <div class="sub-1">
    ------------||--------------
    this is sub-1... template
    <div class="sub2-1">
    this is sub2-1... template
    <div class="sub2-2"></div>
</div>
    <span>hello</span>
    <div class="sub2-3">
    this is sub2-3... template
    <div class="sub3-1">
    this is sub3-1... template
    <div class="sub2-2"></div>
</div>
</div>
</div>
</body>
</html>